src/Compilers/CSharp/Portable/Emitter/Model/PENetModuleBuilder.cs

```diff
@@ -25,12 +25,6 @@ protected override void AddEmbeddedResourcesFromAddedModules(ArrayBuilder<Cci.Ma
             throw ExceptionUtilities.Unreachable;
         }
 
-        internal override SynthesizedAttributeData SynthesizeEmbeddedAttribute()
-        {
-            // Embedded attributes should never be synthesized in modules.
```

Comment:
>// Embedded attributes should never be synthesized in modules. [](start = 12, length = 62)

Are we guarding for this elsewhere?

Reply:
This is now covered below: `EnsureEmbeddedAttributeExists` would throw.

---
In reply to: [215102696](https://github.com/dotnet/roslyn/pull/29180#discussion_r215102696) [](ancestors = 215102696)